manifests/params.pp

```diff
@@ -116,46 +116,54 @@
             '/var/log/httpd/error_log'  => 'apache'
           }
           case $::operatingsystem {
+            'Amazon': {
+              $wodle_openscap_content = {
+                'ssg-centos-6-ds.xml' => {
```

Comment:
Amazon Linux is based on RHEL, maybe use `ssg-rhel-7-ds.xml` instead?

Reply:
RHEL-7 uses systemd; Centos-6 and Amazon do not.  Amazon is a mixture of Centos-6 and RHEL-7, with a four-digit major-version based on the year of release.  Probably there should be an Amazon-specific file, honestly.